### PR TITLE
Add SSD Health CLI utility

### PR DESCRIPTION
**Pull request by andriymoroz-mlnx**
Signed-off-by: Andriy Moroz <c_andriym@mellanox.com>

**- What I did**
Added CLI for SSH Health API

**- How I did it**
utility + "show platform ssdhealth"

**- How to verify it**

**- Previous command output (if the output of a command-line utility has changed)**

**- New command output (if the output of a command-line utility has changed)**
Output examples for different SSD models:
```
root@arc-switch1038:/home/admin# show platform ssdhealth
Device Model : StorFly VSFBM4XC030G-MLX
Health       : 99.635%
Temperature  : 47C

root@arc-switch1038:/home/admin# show platform ssdhealth --verbose
Command: ssdutil -d /dev/sda -v
Device Model : StorFly VSFBM4XC030G-MLX
Firmware     : 0728-000
Serial       : P1T14004420709110047
Health       : 99.635%
Temperature  : 47C
root@arc-switch1038:/home/admin#

```
```
root@mts-sonic-dut:/home/admin# show platform ssdhealth
Device Model : InnoDisk Corp. - mSATA 3ME
Health       : 30.067%
Temperature  : 0C

root@mts-sonic-dut:/home/admin# show platform ssdhealth --verbose
Command: ssdutil -d /dev/sda -v
Device Model : InnoDisk Corp. - mSATA 3ME
Firmware     : S140714
Serial       : 20160404AA140000004B
Health       : 30.067%
Temperature  : 0C
```
-->

### CODE REVIEW

**Comment by liat-grozovik** [comment]
retest default please

File: ssdutil/main.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        try:
+            from sonic_platform_base.sonic_ssd.ssd_generic import SsdUtil
+        except ImportError as e:
+            syslog_msg(syslog.LOG_ERR, "Failed to import default SsdUtil. Error: {}".format(str(e)), True)
+            raise e
+
+    return SsdUtil(diskdev)
+
+# ==================== Entry point ====================
+def cli():
```
Comment (jleveque) [inline ssdutil/main.py]: Suggest to rename the Click entrypoint to match the command name `def ssdutil():`

Comment (andriymoroz-mlnx) [reply ssdutil/main.py]: renamed

**Comment by andriymoroz-mlnx** [comment]
Last update skips health and temperature units if value is N/A
e.g.
```
Before:
root@arc-switch1038:/home/admin# show platform ssdhealth
Device Model : StorFly VSFBM4XC030G-MLX
Health       : 99.575%
Temperature  : 47C

root@arc-switch1038:/home/admin# show platform ssdhealth /dev/sdb
Device Model : N/A
Health       : N/A%
Temperature  : N/AC

After:
root@arc-switch1038:/home/admin# show platform ssdhealth
Device Model : StorFly VSFBM4XC030G-MLX
Health       : 99.575%
Temperature  : 43C

root@arc-switch1038:/home/admin# show platform ssdhealth /dev/sdb
Device Model : N/A
Health       : N/A
Temperature  : N/A

```

**Pull request merged**